src/pretix/api/views/order.py

```diff
@@ -2030,3 +2031,61 @@ class BlockedSecretViewSet(viewsets.ReadOnlyModelViewSet):
 
     def get_queryset(self):
         return BlockedTicketSecret.objects.filter(event=self.request.event)
+
+
+with scopes_disabled():
+    class TransactionFilter(FilterSet):
+        order = django_filters.CharFilter(field_name='order', lookup_expr='code__iexact')
+        event = django_filters.CharFilter(field_name='order__event', lookup_expr='slug__iexact')
```

Comment:
Should we subclass the TransactionFilter for OrganizerTransactionViewSet? Currently one can build useless queries by adding ?event=abc to the URL even when using the event-based transactions-URL. I think it’s okay to leave it like that, but just wanted to make sure, we are aware of it.

Reply:
I don't think there's any harm in keeping this, I did not document it so we can remove it, but it's also quite useless anyways.